src/base/misc/disks.c

```diff
@@ -581,7 +582,7 @@ static void hdisk_auto(struct disk *dp)
       error("Hmm... BLKSSZGET failed (not fatal): %s\n", strerror(errno));
       sector_size = SECTOR_SIZE;
     }
-    dp->num_secs += sector_size - 1;	/* round up */
+    /* dp->num_secs += sector_size - 1;	/* round up */ /* round down! */
```

Comment:
disks.c: In function 'hdisk_auto':
disks.c:594:41: warning: "/*" within comment [-Wcomment]